api/application/client.go

```diff
@@ -621,6 +648,8 @@ func (c *Client) DestroyConsumedApplication(saasNames ...string) ([]params.Error
 		index = append(index, i)
 		args.Applications = append(args.Applications, params.DestroyConsumedApplicationParams{
 			ApplicationTag: names.NewApplicationTag(name).String(),
+			Force: in.Force,
```

Comment:
Please only add Force and MaxWait if the BestAPIVersion is > 9. Previous versions do not need them.

Reply:
Will do. I left this in because they would default to 0-values, so didn't think it was important.